proto/sensei.proto

```diff
@@ -292,6 +292,7 @@ message KeysendResponse {}
 message CreateInvoiceRequest {
     uint64 amt_msat = 1;
     string description = 2;
+    optional uint32 expiry = 3;
```

Comment:
not sure if we do this other places with time but I personally prefer to know the units similar to amt_msat.  When I look at expiry the first thing I'll ask is if it's in milliseconds or seconds.  how do you feel about naming it expiry_secs?

Reply:
Sounds great to me 👌